Makefile

```diff
@@ -140,12 +140,18 @@ start-mongodb4-single:
 start-mongodb5-single:
 	docker-compose -f docker-compose.yml up -d --remove-orphans mongo5-single
 
-start-mongodb4:
-	docker-compose -f docker-compose.yml up -d --remove-orphans mongo4
+start-mongodb42-single:
```

Comment:
Throughtout the entire PR, I'd prefer one of the two conventions we use: `4_2` or `4-2` (see other commands)